Source/ASImageNode.mm

```diff
@@ -301,8 +299,8 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
   drawParameters->_contentsScale = _contentsScaleForDisplay;
   drawParameters->_backgroundColor = self.backgroundColor;
   drawParameters->_contentMode = self.contentMode;
-  drawParameters->_cropEnabled = _cropEnabled;
-  drawParameters->_forceUpscaling = _forceUpscaling;
+  drawParameters->_cropEnabled = self.cropEnabled;
+  drawParameters->_forceUpscaling = self.forceUpscaling;
```

Comment:
Why not using the `_imageNodeFlags` for both in here? We are already having the lock.

Reply:
Aha! I see what you mean.